src/platform/packages/shared/response-ops/rule_form/src/rule_form.tsx

```diff
@@ -178,26 +182,27 @@ export const RuleForm = <MetaData extends RuleTypeMetaData = RuleTypeMetaData>(
     docLinks,
     ruleTypeRegistry,
     actionTypeRegistry,
+    fieldsMetadata,
```

Comment:
Prettier or something in Windsurf rearranged these dependencies, I don't know why.